### PR TITLE
Update BambuStudio_es.po

### PR DESCRIPTION
**Pull request by mainmind83**
Spanish

### CODE REVIEW

File: bbl/i18n/es/BambuStudio_es.po [hunk 1]
```diff
@@ -866,16 +865,16 @@ msgid "Clone"
...
 msgid "Center"
 msgstr ""
 
 msgid "Edit Process Settings"
-msgstr ""
+msgstr "Editar configuración del proceso"
 
 msgid "Edit in Parameter Table"
-msgstr ""
+msgstr "Edtiar en tabla de parámetros"
```
Comment (jwise) [inline bbl/i18n/es/BambuStudio_es.po:877]: ¿"Editar" en vez de "Edtiar"?

Comment (mainmind83) [reply bbl/i18n/es/BambuStudio_es.po]: Yes @jwise  ;)

**Comment by mainmind83** [comment]
Waiting... @jwise ;D

**Comment by jwise** [comment]
@mainmind83 hey, I'm just an open source contributor who took high school spanish -- I don't work for Bambu :)

tagging @lanewei120 perhaps instead?

**Comment by lanewei120** [comment]
@mainmind83 
thanks for doing this
we will ask our Spanish translator to review it

**Comment by mainmind83** [comment]
Main menu has some translation errors ;D thanks @lanewei120

**Comment by lanewei120** [comment]
thanks very much for doing this
@mainmind83 

I am sorry the time cost is long for our Spanish translator is part time
and now he finished review

Could you rebase to the master and solve the conflicts again?
then we can merge it

currently it can not be merged due to conflicts
image

**Comment by mainmind83** [comment]
New branch: https://github.com/bambulab/BambuStudio/pull/1533

**Comment by mainmind83** [comment]
I have created a new translation for the current version from this branch @lanewei120

**Pull request closed without merging**